app/models/school.rb

```diff
@@ -48,13 +48,15 @@ class School < ApplicationRecord
   include PgSearch::Model
 
   has_many :memberships, as: :organisation
-  has_many :users, through: :memberships
   has_many :mentors
 
   validates :urn, presence: true
   validates :urn, uniqueness: { case_sensitive: false }
   validates :name, presence: true
 
+  scope :placements_service, -> { where placements_service: true }
+  scope :claims_service, -> { where claims_service: true }
```

Comment:
It might be a good idea to re-introduce the scope tests that I removed in #157.

https://github.com/DFE-Digital/itt-mentor-services/blob/2e0369e234ed49b8ffb7a18c233284b46bde6cac/spec/models/school_spec.rb#L62-L78

I realise that we do still [implicitly test this functionality](https://github.com/DFE-Digital/itt-mentor-services/blob/f8c00ef494d491f36c47f754816a41a80afd1686/spec/models/placements/school_spec.rb#L50-L58) on the namespaced classes. So this isn't technically un-covered. But would be nice to have an explicit test since it's part of `School`'s public interface.

Reply:
Doh, of course!